grails-app/controllers/bie/index/UrlMappings.groovy

```diff
@@ -58,7 +58,7 @@ class UrlMappings {
 
         "/logout/logout" (controller: 'misc', action: 'logout')
 
-        "/"(view: "/index")
+        "/"(redirect: "/openapi")
```

Comment:
Why do we need this?

Reply:
This tires to maintain the behaviour of the current bie-index's index `/` [page](https://bie-ws.ala.org.au/ws/) which displays the swagger UI under the `/index` path. New changes replace the old swagger iplementation with openapi-1.0 for api spec definition and this change displayes the `/opeanapi` page for the `/`

Reply:
ah ok